Can't install simple-stack-form from npm

Project with Astro 4.0.*
Command: `npm i simple-stack-form`

```
landing-template git:(master) ✗ npm i simple-stack-form
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: landing-template@0.0.1
npm ERR! Found: astro@4.0.3
npm ERR! node_modules/astro
npm ERR!   astro@"^4.0.3" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer astro@"^3.6.0" from simple-stack-form@0.0.4
npm ERR! node_modules/simple-stack-form
npm ERR!   simple-stack-form@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!

```

--- a/packages/form/package.json
+++ b/packages/form/package.json
@@ -21,9 +21,9 @@
   "author": "bholmesdev",
   "license": "MIT",
   "devDependencies": {
-    "astro": "^3.6.0"
+    "astro": "^4.0.0"
   },
   "peerDependencies": {
-    "astro": "^3.6.0"
+    "astro": "^3.6.0 || ^4.0.0"
   }
 }

